src/scss/main.scss

```diff
@@ -4,3 +4,176 @@
 .create-schedule {
   display: none;
 }
+
+.nav{
+  margin:0px;
+}
+
+.headline{
+  background-color:#80D8FF;
+  color:white;
+  width:300px;
+  height:70px;
+  text-align:center;
+  padding-top:20px;
+  font-size:20px;
+}
+
+body, html{
+font-family: $helvetica;
+margin:0px;
+}
+
+.datepicker{
+  font-family: $helvetica;
+}
+
+a{
+  text-decoration:none;
+}
+
+header{
+  height:80px;
+}
+.logo{
+  float:left;
+}
+
+header li{
+  color:red;
+  font-size:25px;
+  display:inline-block;
+  padding-right:40px;
+  float:right;
+}
+
+form{
+  border-left:1px solid black;
+  border-right:1px solid black;
+  border-bottom:1px solid black;
+  margin:50px;
+  height:800px;
+}
+
+form ul{
+  list-style-type: none;
+  padding-left:0px;
+  margin-bottom:50px;
+}
+
+.create-email input{
+  width:100%;
+  height:40px;
+}
+
+.schedule-send{
+  height:500px;
+  height:800px;
+}
+
+.schedule-send input{
+  width:100%;
+  height:40px;
+}
+.schedule-send li{
+  list-style-type:none;
+}
+.create-email h2{
```

Comment:
For organization, I like to keep similar selectors near each other, there is a `.create-email` selector on line 64, I would move this up to be right below, so they are grouped together.